frontend/src/utils/fillPatterns.ts

```diff
@@ -0,0 +1,116 @@
+    case FillPatternType.CrossHatch:
+      return getCrosshatchFillPattern(color);
+    case FillPatternType.Points:
+      return generatePointsPatternAsImage(color);
+
+    default:
+      return undefined;
+  }
+}
+
+export function getHatchDownFillPattern(color: string) {
+  const svgPattern = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+  svgPattern.setAttribute('width', '20');
+  svgPattern.setAttribute('height', '20');
+
+  const line = document.createElementNS('http://www.w3.org/2000/svg', 'line');
+  line.setAttribute('x1', '0');
+  line.setAttribute('y1', '0');
+  line.setAttribute('x2', '20');
+  line.setAttribute('y2', '20');
+  line.setAttribute('stroke', color);
+  line.setAttribute('stroke-width', '1');
+  svgPattern.appendChild(line);
+
+  const patternImage = new Image();
+  patternImage.src =
+    'data:image/svg+xml;base64,' + btoa(new XMLSerializer().serializeToString(svgPattern));
+
+  return patternImage;
+}
+
+export function getHatchUpFillPattern(color: string) {
+  const svgPattern = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+  svgPattern.setAttribute('width', '20');
+  svgPattern.setAttribute('height', '20');
+
+  const line = document.createElementNS('http://www.w3.org/2000/svg', 'line');
+  line.setAttribute('x1', '0');
+  line.setAttribute('y1', '20');
+  line.setAttribute('x2', '20');
+  line.setAttribute('y2', '0');
+  line.setAttribute('stroke', color);
+  line.setAttribute('stroke-width', '1');
+  svgPattern.appendChild(line);
+
+  const patternImage = new Image();
+  patternImage.src =
+    'data:image/svg+xml;base64,' + btoa(new XMLSerializer().serializeToString(svgPattern));
+
+  return patternImage;
+}
+
+export function getCrosshatchFillPattern(color: string) {
+  const svgPattern = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+  svgPattern.setAttribute('width', '20');
+  svgPattern.setAttribute('height', '20');
+
+  const line1 = document.createElementNS('http://www.w3.org/2000/svg', 'line');
+  line1.setAttribute('x1', '0');
+  line1.setAttribute('y1', '0');
+  line1.setAttribute('x2', '20');
+  line1.setAttribute('y2', '20');
+  line1.setAttribute('stroke', color);
+  line1.setAttribute('stroke-width', '1');
+  svgPattern.appendChild(line1);
+
+  const line2 = document.createElementNS('http://www.w3.org/2000/svg', 'line');
+  line2.setAttribute('x1', '0');
+  line2.setAttribute('y1', '20');
+  line2.setAttribute('x2', '20');
+  line2.setAttribute('y2', '0');
+  line2.setAttribute('stroke', color);
+  line2.setAttribute('stroke-width', '1');
+  svgPattern.appendChild(line2);
+
+  const patternImage = new Image();
+  patternImage.src =
+    'data:image/svg+xml;base64,' + btoa(new XMLSerializer().serializeToString(svgPattern));
+
+  return patternImage;
+}
+
+export function generatePointsPatternAsImage(color: string) {
+  const svgString = `<svg width="100" height="100" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is there a specific reason why you hard coded this svg data?
I would prefer it if you saved it to its own file, importing it like we do with our icons.

This would make it easier to edit fill patterns in the future, especially for non-technical team members such as @yvonnemarkl.

Reply:
you are right, i also don't really like the current solution.
 i had some struggles with importing the svg and then create a html image as konva is expecting. 
However, I will take a further look into this.